source/administration/config-settings.rst

```diff
@@ -2054,6 +2054,14 @@ This filter is defaulted to ``(|(objectClass=group)(objectClass=groupOfNames)(ob
 | This feature's ``config.json`` setting is ``"GroupFilter": ""`` with string input.                                                                                   |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Guest Filter
+^^^^^^^^^^^^^^
```

Comment:
@amyblais Justine has these documented in this open PR: https://github.com/mattermost/docs/pull/3213